src/styles/app.css

```diff
@@ -0,0 +1,17 @@
+.app {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: var(--white-space);
+}
+
+header {
+  max-width: 300px;
+  width: 100%;
```

Comment:
Since `header` is a block level element, its width is implicitly 100% so you can remove this